content/en/docs/concepts/configuration/configmap.md

```diff
@@ -16,33 +16,45 @@ or use additional (third party) tools to keep your data private.
 {{< /caution >}}
 
 
-
 <!-- body -->
 ## Motivation
 
 Use a ConfigMap for setting configuration data separately from application code.
 
 For example, imagine that you are developing an application that you can run on your
 own computer (for development) and in the cloud (to handle real traffic).
-You write the code to
-look in an environment variable named `DATABASE_HOST`. Locally, you set that variable
-to `localhost`. In the cloud, you set it to refer to a Kubernetes
-{{< glossary_tooltip text="Service" term_id="service" >}} that exposes the database
-component to your cluster.
-
+You write the code to look in an environment variable named `DATABASE_HOST`.
+Locally, you set that variable to `localhost`. In the cloud, you set it to
+refer to a Kubernetes {{< glossary_tooltip text="Service" term_id="service" >}}
+that exposes the database component to your cluster.
 This lets you fetch a container image running in the cloud and
 debug the exact same code locally if needed.
 
+A ConfigMap is not designed to hold large chunks of data. The data stored in a
+ConfigMap object cannot exeed 1 MiB. If you need to store settings that are
```

Comment:
nit: can remove "object" after ConfigMap?
nit: could drop the clause, "for example" at end of line 36.